Assets/Scripts/UI/HUD/ActivePlayer.cs

```diff
@@ -0,0 +1,24 @@
+using UnityEngine;
+using UnityEngine.UI;
+using OperationBlackwell.Core;
+
+namespace OperationBlackwell.UI {
+	public class ActivePlayer : MonoBehaviour {
+		[SerializeField] private Image activePlayer_;
+
+		private void Update() {
```

Comment:
Please do this with events, the Update has a lot of overhead if the player does not change that frame.

Reply:
Done